test/test_alexa_app_customslot.js

```diff
@@ -10,6 +10,30 @@ chai.config.includeStack = true;
 import * as Alexa from '..';
 
 describe("Alexa", function() {
+
+  context("#alexa.request.slot", function () {
+    var testReq = mockHelper.load("intent_request_airport_info.json");
+    
+    beforeEach(function () {
+      testReq = mockHelper.load("intent_request_airport_info.json");
+    });
+
+    afterEach(() => {
+      testReq = null;
+    });
+
+    it("get slot value if correct", function () {
+      var req = new Alexa.request(testReq);
+      expect(req.slot("AirportCode", "SEA")).to.eql("JFK");
+    });
+
+    it("gets defaultValue if slot value is undefined", function () {
+      testReq.request.intent.slots.AirportCode.value = undefined;
+      var req = new Alexa.request(testReq);
+      expect(req.slot("AirportCode", "SEA")).to.eql("SEA");
+    });
+  });
+  
   describe("app", function() {
     var testApp = new Alexa.app("testApp");
 
```

Comment:
@dblock was just following the convention below. It seems this is a work-around for `dtslint` causing issues if you just declare a variable placeholder say `var testApp` without assigning it.

Reply:
@dblock - reference to your comment on line 18

Reply:
by just declaring ` var testReq;` ts rightfully complains `Variable 'testReq' implicitly has an 'any' type.`. But in a JS file you cant declare a type to silence the error

Reply:
I would advocate for changing `"noImplicitAny": true,` to `false` inside the `test/tsconfig.json` file

Reply:
Feel free to PR separately whatever you see fit, we're splitting hairs ;)